src/Stack/Types/Config.hs

```diff
@@ -1041,9 +1049,15 @@ configMonoidIgnoreRevisionMismatchName = "ignore-revision-mismatch"
 configMonoidColorWhenName :: Text
 configMonoidColorWhenName = "color"
 
+configMonoidColorWhenAlternativeName :: Text
+configMonoidColorWhenAlternativeName = "colour"
+
 configMonoidStylesName :: Text
 configMonoidStylesName = "stack-colors"
 
+configMonoidStylesAlternativeName :: Text
+configMonoidStylesAlternativeName = "stack-colours"
+
```

Comment:
Likewise here (`s/Alternative/British/`)

Reply:
Done - I now use `US` and `GB` consistently.